src/transformers/modeling_flax_utils.py

```diff
@@ -664,13 +664,14 @@ def from_pretrained(
                     # Load from a sharded Flax checkpoint
                     archive_file = os.path.join(pretrained_model_name_or_path, FLAX_WEIGHTS_INDEX_NAME)
                     is_sharded = True
-                # At this stage we don't have a weight file so we will raise an error.
```

Comment:
Not sure why this comment is deleted as an error is raised after this stage (just different ones depending on the if). It should be left here.

Reply:
oh indded